css/style.css

```diff
@@ -288,5 +288,30 @@ footer p {
   font-weight: 600;
 }
 
+.hamburger {
+  display: none;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
I'd also recommend `blockquote` to have a `max-width: 300px` rather than a `width`. Same goes for `#benefits p, #benefits ul`.

Also recommend using 
```
img {
 max-width: 100%;
}
```
You can do that as a global rule (not scoped to a media query). That way the images never grow beyond their container allows. Right now they cause horizontal scrolling when in small / mobile mode

#sidebar should be 100% for smaller screens

Recommend removing `margin-right: 20px` on `#posts` when in small screen